doc/ref_model/chapters/chapter02.md

```diff
@@ -75,6 +75,7 @@ The list of, most likely to be virtualised,  Network Functions below, covering a
     - UDR: Unified Data Repository
     - NRF:  Network Repository Function
     - NEF:  Network Exposure Function
+    - CHF/CCS:  Charging Function / Converged Charging System
```

Comment:
please split CHF and CCS into their own separate rows.

Reply:
I can do that, but CHF is a part of the CCS,  CHF interacts on the SBA, the CCS does not.  To me it makes more sense to keep together, or just drop CCS.

Reply:
Agree that they are related -- but we have other related functions listed separately.  

It is your decision to make. 

BTW you will also need to add these to the abbreviations list